test/quantization/test_qat.py

```diff
@@ -455,7 +455,7 @@ def test_qat_4w_linear(self):
     # TODO: remove once we fix int4 error: https://github.com/pytorch/ao/pull/517
     @unittest.skipIf(TORCH_VERSION_AFTER_2_5, "int4 doesn't work for 2.5+ right now")
     def test_qat_4w_quantizer(self):
-        from torchao.quantization.prototype.qat import Int4WeightOnlyQATQuantizer
+        from torchao.quantization.prototype.qat.api import Int4WeightOnlyQATQuantizer
```

Comment:
This renaming seems worse and I don't believe you need it since `Int4WeightOnlyQATQuantizer` is part of `qat/__init__.py`